sensing/imu_corrector/src/gyro_bias_estimation_module.hpp

```diff
@@ -11,33 +11,33 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
 #ifndef GYRO_BIAS_ESTIMATION_MODULE_HPP_
 #define GYRO_BIAS_ESTIMATION_MODULE_HPP_
 
-#include <geometry_msgs/msg/vector3.hpp>
+#include <geometry_msgs/msg/pose_stamped.hpp>
+#include <geometry_msgs/msg/vector3_stamped.hpp>
 
 #include <deque>
+#include <utility>
+#include <vector>
 
 namespace imu_corrector
 {
 class GyroBiasEstimationModule
 {
 public:
-  GyroBiasEstimationModule(
-    const double velocity_threshold, const double timestamp_threshold,
-    const size_t data_num_threshold);
-  geometry_msgs::msg::Vector3 get_bias() const;
-  void update_gyro(const double time, const geometry_msgs::msg::Vector3 & gyro);
-  void update_velocity(const double time, const double velocity);
+  explicit GyroBiasEstimationModule(const size_t data_num_threshold);
+  void update_bias(
+    const std::vector<geometry_msgs::msg::PoseStamped> & pose_list,
+    const std::vector<geometry_msgs::msg::Vector3Stamped> & gyro_list);
+  geometry_msgs::msg::Vector3 get_bias_base_link() const;
+  geometry_msgs::msg::Vector3 get_bias_std() const;
```

Comment:
Would you erase this function as it is currently not used in gyro_bias_estimator?

Reply:
Erased in
https://github.com/autowarefoundation/autoware.universe/pull/5259/commits/482b3ccb6be56ecc064881fd625f48348c4d718b